src/com/teamcautionrobotics/robot2018/Robot.java

```diff
@@ -63,23 +75,38 @@ public void robotInit() {
 
         commandFactory = new CommandFactory(driveBase);
 
-        missionScriptMission = new MissionScriptMission("Mission Script Mission", missionScriptPath,
-                commandFactory);
+        startingPositionChooser = new SendableChooser<>();
+        startingPositionChooser.addDefault("Center", StartingPosition.CENTER);
+        startingPositionChooser.addObject("Left", StartingPosition.LEFT);
+        startingPositionChooser.addObject("Right", StartingPosition.RIGHT);
+        SmartDashboard.putData("Starting Position Select", startingPositionChooser);
+
+        autoObjectiveChooser = new SendableChooser<>();
+        autoObjectiveChooser.addDefault("Switch", AutoObjective.SWITCH);
+        autoObjectiveChooser.addObject("Scale", AutoObjective.SCALE);
+        autoObjectiveChooser.addObject(AutoObjective.SWITCH_OR_SCALE.toString(), AutoObjective.SWITCH_OR_SCALE);
+        autoObjectiveChooser.addObject("Auto line", AutoObjective.AUTO_LINE);
+        autoObjectiveChooser.addObject("Do nothing", AutoObjective.DO_NOTHING);
+        SmartDashboard.putData("Auto Objective Select", autoObjectiveChooser);
 
         missionChooser = new SendableChooser<>();
 
-        missionChooser.addObject("Do not use -- Mission Script", missionScriptMission);
-        missionChooser.addDefault("Do Nothing Mission", new Mission("Do Nothing Mission"));
+        missionScriptMission = new MissionScriptMission("Mission Script Mission", missionScriptPath,
+                commandFactory);
+        missionChooser.addDefault("Do not use -- Mission Script", missionScriptMission);
         SmartDashboard.putData("Autonomous Mode Select", missionChooser);
```

Comment:
The "Autonomous Mode Select"ion is not used. I believe this was going to be used to select whether the mission selection logic runs.